test_util/azure_test_driver.py

```diff
@@ -55,20 +71,12 @@ def run():
     # tenant_id = os.environ.get('AZURE_TENANT_ID')
     # client_id = os.environ.get('AZURE_CLIENT_ID')
     # client_secret = os.environ.get('AZURE_CLIENT_SECRET')
-    group_name = 'tesing' + ''.join(random.choice('01234567890abcdef') for n in range(10))
+    group_name = 'testing' + ''.join(random.choice('01234567890abcdef') for n in range(10))
     deployment_name = 'deployment{}'.format(uuid.uuid4().hex)
 
     rmc = ResourceManagementClient(credentials, subscription_id)
 
     template_parameters = get_env_params()
-    if template_parameters.get('numberOfPrivateSlaves'):
-        assert template_parameters['numberOfPrivateSlaves']['value'] >= 2, 'Test requires at least 2 private slaves!'
```

Comment:
Intentionally removed, since I want the test driver to be more generic and I don't think this logic belongs here. In general, I think it should be easy for us to create CI jobs for a variety of cluster sizes and types... and adding this logic here just limits our test options.